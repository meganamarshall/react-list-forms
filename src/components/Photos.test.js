import React from 'react';
import { shallow } from 'enzyme';
import Photos from './Photos';
import candyByColor from '../assets/candyByColor.jpg';
import gettyImages from '../assets/gettyimages-649680072.jpg';
import images from '../assets/images.jpeg';
import othercandy from '../assets/othercandy.jpg';

describe('photos component', () => {
  it('renders the photos', () => {
    const photoArray = [
      candyByColor,
      gettyImages,
      images,
      othercandy
    ];
    const wrapper = shallow(<Photos photos={photoArray} />);
    expect(wrapper).toMatchSnapshot();
  });
});
