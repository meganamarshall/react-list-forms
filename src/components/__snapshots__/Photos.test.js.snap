// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photos component renders the photos 1`] = `
<ul>
  <li
    key="/path/image"
  >
    <Photo
      photo="/path/image"
    />
  </li>
  <li
    key="/path/image"
  >
    <Photo
      photo="/path/image"
    />
  </li>
  <li
    key="/path/image"
  >
    <Photo
      photo="/path/image"
    />
  </li>
  <li
    key="/path/image"
  >
    <Photo
      photo="/path/image"
    />
  </li>
</ul>
`;
